tests/baselines/reference/convertToAsyncFunction/convertToAsyncFunction_catchBlockUniqueParams.js

```diff
@@ -0,0 +1,19 @@
+// ==ORIGINAL==
+
+function /*[#|*/f/*|]*/() {
+	return Promise.resolve().then(x => 1).catch(x => "a").then(x => !!x); 
+} 
+
+// ==ASYNC FUNCTION::Convert to async function==
+
+async function f() {
+	let x_2;
```

Comment:
>x_2 [](start = 5, length = 3)

Nit: It's kind of weird that the catch parameter got "1".

Reply:
That's just because we rename from left to right in the pre-fixed form. Not sure if that's worth fixing at this point.